### ISSUE TITLE
KAFE MACHINE: Classification

### ISSUE THREAD

**Issue by joshmessi10** [joshmessi10/KAFE-Reloaded#47]
Regresión Logística

- Sigmoid
- Loss (cross-entropy)
- Gradient Descent
